components/script/dom/rtcpeerconnection.rs

```diff
@@ -503,7 +503,7 @@ impl RTCPeerConnectionMethods<crate::DomTypeHolder> for RTCPeerConnection {
         config: &RTCConfiguration,
     ) -> Fallible<DomRoot<RTCPeerConnection>> {
         Ok(RTCPeerConnection::new(
-            window,
+            &window.global(),
```

Comment:
We can change the argument for RTCPeerConnection::new, too, since it's only exposed in Window globals: https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection